doc/source/developer_information/developer_reference/revent.rst

```diff
@@ -26,10 +28,12 @@ Only the run stage is mandatory, the remaining stages will be replayed if a
 recording is present otherwise no actions will be performed for that particular
 stage.
 
-For instance, to add a custom revent files for a device named "mydevice" and
-a workload name "myworkload", you need to add the revent files to the directory
-``/home/$WA_USER_HOME/dependencies/myworkload/revent_files`` creating it if
-necessary. ::
+All your custom revent files should reside at
+``'$WA_USER_DIRECTORY/.workload_automation/dependencies/WORKLOAD NAME/'``. So
```

Comment:
Doesn't `$WA_USER_DIRECTORY` already included the `.workload_automation` part?

Reply:
Yes thank you, fixed the references.